addons/im_livechat/models/digest.py

```diff
@@ -0,0 +1,46 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import fields, models, api
+
+
+class Digest(models.Model):
+    _inherit = 'digest.digest'
+
+    kpi_im_livechat_rating = fields.Boolean('% of Happiness')
+    kpi_im_livechat_rating_value = fields.Float(digits=(16, 2), compute='_compute_kpi_livechat_rating_value')
+    kpi_im_livechat_conversations = fields.Boolean('Conversations handled')
+    kpi_im_livechat_conversations_value = fields.Integer(compute='_compute_kpi_livechat_conversations_value')
+    kpi_im_livechat_response = fields.Boolean('Time to answer(sec)', help="Time to answer the user in second.")
+    kpi_im_livechat_response_value = fields.Float(compute='_compute_kpi_livechat_response_value')
```

Comment:
I think you can remove here 'im_' from the fields name, so that the computed methods follow the rule '_compute_<field_name>'. (because doing the opposite by adding '_im' in the compute names will be longer and because everyone know livechat = im_livechat)